packages/server/utils/OpenAIServerManager.ts

```diff
@@ -221,9 +221,11 @@ class OpenAIServerManager {
       })
 
       const templateResponse = (response.choices[0]?.message?.content?.trim() as string) ?? null
-      return JSON.parse(templateResponse) as AITemplateSuggestion
+      const parsedResponse = JSON5.parse(templateResponse)
```

Comment:
Just curious, why replaced with JSON5?

Reply:
3.5-turbo doesn't have the JSON parse feature that GPT-4 has, so we need to parse it ourselves